integration/test_auth.py

```diff
@@ -177,6 +177,27 @@ def new_user_with_params():
     yield user
     auth.delete_user(user.uid)
 
+@pytest.fixture
+def imported_user_with_params():
+    random_id, email = _random_id()
+    phone = _random_phone()
+    user = auth.ImportUserRecord(
+        uid=random_id,
+        email=email,
+        phone_number=phone,
+        display_name='Random User',
+        photo_url='https://example.com/photo.png',
+        user_metadata=auth.UserMetadata(100, 150),
```

Comment:
I couldn't figure out what 100,150 were without looking up UserMetadata. Named parameters would make this more obvious, i.e. 

`auth.UserMetadata(creation_timestamp=100, last_sign_in_timestamp=150)`

Optional.